packages/xo-web/src/common/intl/messages.js

```diff
@@ -1764,21 +1764,21 @@ const messages = {
   refresh: 'Refresh',
   upgrade: 'Upgrade',
   downgrade: 'Downgrade',
-  noUpdaterCommunity: 'No updater available for Community Edition',
+  noUpdaterCommunity: 'One Click updates are not available for the Community Edition',
   considerSubscribe:
-    'Please consider subscribing and trying it with all the features for free during 15 days on {link}.',
+    'Please consider subscribing and trying XOA with all the features for free for 15 days by going to {link}.',
   noUpdaterWarning:
-    'Manual update could break your current installation due to dependencies issues, do it with caution',
+    'Manually updating your installation could break due to dependency issues, upgrade with caution',
   currentVersion: 'Current version:',
   register: 'Register',
   editRegistration: 'Edit registration',
   trialRegistration:
-    'Please, take time to register in order to enjoy your trial.',
+    'Please, take time to register and order enjoy your 15 day trial.',
```

Comment:
this should probably be "Please, take the time to register in order to enjoy your 15 day trial" as we don't mean to literally order anything, just that registering is required to begin a trial. The rest looks good :)

Reply:
Changed to match the recommendation. No coffee at the time of the edit. . .